### DISCUSSION TITLE
Can I use Vuetify for Angular +6?

### DISCUSSION THREAD

**Discussion by ghost** [vuetifyjs/vuetify#12795] [General] [answered]
Can I use Vuetify for Angular?

Hello Everybody,

Can I use Vuetify or any themes similar for Angular?

**Comment by DRoet** [comment, marked as answer]
No, Vuetify is Vue specific, for Angular you will have to use a Angular framework like https://material.angular.io/

**Reply by ghost** [reply]
I know but Angular Material need more customized. 
I saw material-ui.com but this material for react. 
I still search on theme or material like Vuetify for Angular.

**Reply by djcaesar9114** [reply]
You could switch to Vue, maybe, as I did :)

**Reply by ghost** [reply]
I use Vue Js and Angular Js at the same time in multiple projects, but I need a UI library for Angular like Vuetify, not the same but similar for it.
Thanks.
:D
